### PR TITLE
Enable webflux tests for backend

### PR DESCRIPTION
**Pull request by gzsombor**
Webflux tests run only when explicitly asked for, which make these tests stale, and invalid.
---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by murdos** [comment]
👍

**Comment by pascalgrimaud** [comment]
I was the one who disabled it because it was simply broken

@gzsombor : 
- how can webflux be ignored when doing a pull request, which is not related at all to webflux ?
- it was disabled because it's broken since 1 month or more -> see https://github.com/jhipster/jhipster/pull/872#issuecomment-705997172

**Review by mshima — APPROVED** [review]

**Comment by mshima** [comment]
> I was the one who disabled it because it was simply broken

@pascalgrimaud we fix then we break.
If webflux is in main repository, we must have tests enabled for it, otherwise we should move it to a blueprint.

At this PR we fixed https://github.com/jhipster/generator-jhipster/pull/12536 
This PR broke it again https://github.com/jhipster/generator-jhipster/pull/12507

Now we have a bounty greater than the original PR to fix webflux.
I want to add basic microservice test too. Once the scripts PR is merged.
We could move some tests to microservice, so ms1 tests db1, ms2 tests db2, in the same test.

**Comment by murdos** [comment]
>     * it was disabled because it's broken since 1 month or more -> see [jhipster/jhipster#872 (comment)](https://github.com/jhipster/jhipster/pull/872#issuecomment-705997172)

@pascalgrimaud : I think it's the other way around: it's broken since 1 month or more because there are no CI builds that detect that webflux has been broken possibly by PRs not related to webflux 😄

**Comment by pascalgrimaud** [comment]
Ok, Let's give a new chance to webflux :)

**Comment by mshima** [comment]
@pascalgrimaud IMO we can merge as it is now.
An alternative is to integrate into https://github.com/jhipster/generator-jhipster/pull/12688 since it should fix webflux tests.
Your call.

**Comment by pascalgrimaud** [comment]
Yes, feel free to merge it, if you feel confident

**Comment by DanielFran** [comment]
Probably it started when introducing the partial updates

**Pull request merged**
